test/test_nestedtensor.py

```diff
@@ -2844,23 +2848,21 @@ def grad_test_func(a, b, c):
 # We can probably parametrizing existing tests instead of having a separate
 # test class as we begin to support more ops. Also maybe rewrite with OpInfos.
 class TestNestedTensorSubclass(TestCase):
+    def _get_list_for_jagged_tensor(self, nested_size, requires_grad=True):
+        Ds = nested_size[1:]
+        out = []
+        for s in nested_size[0]:
+            out.append(
+                torch.randn(s, *Ds, requires_grad=requires_grad, dtype=torch.float64)
+            )
+        return out
+
     def _get_example_tensor_lists(self):
         return [
             # (B, *, D) with B=4
-            [
-                torch.randn(2, 5),
-                torch.randn(3, 5),
-                torch.randn(4, 5),
-                torch.randn(6, 5)
-            ],
+            self._get_list_for_jagged_tensor(((2, 3, 4, 6), 5)),
```

Comment:
is this going to work with the changes in #112304 that give `requires_grad=True` to some of the elements inside?

Reply:
Good catch, yeah looks like this would need to be updated.

Reply:
I ended up just changing it back.